libfreerdp/codec/rfx.c

```diff
@@ -966,10 +966,9 @@ static BOOL rfx_process_message_tileset(RFX_CONTEXT* context, RFX_MESSAGE* messa
 			WaitForThreadpoolWorkCallbacks(work_objects[i], FALSE);
 			CloseThreadpoolWork(work_objects[i]);
 		}
-
-		free(work_objects);
-		free(params);
 	}
+	free(work_objects);
+	free(params);
```

Comment:
work_objects and params are only allocated if threads are used. The free here doesn't hurt but wasn't a leak either before imho.